db/migrate/20230813203012_fix_canadian_french_locale.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class FixCanadianFrenchLocale < ActiveRecord::Migration[7.0]
+  class User < ApplicationRecord
+    # Dummy class, to make migration possible across version changes
+  end
+
+  disable_ddl_transaction!
+
+  def up
+    User.where(locale: 'fr-QC').find_in_batches do |users|
```

Comment:
Could this just be one SQL update, instead of iterating through user objects?

Reply:
Done. I don't know how to test the change though, since I've already run the migration. I find the Ruby documentation too hard to navigate on this topic, so I'm basically just frankencoding these from older migrations.

Reply:
Alright, I can write some migration tests.

Reply:
Oops, this should be `in_batches` and not `find_in_batches`!